charts/cass-operator/values.yaml

```diff
@@ -79,9 +79,9 @@ tolerations: []
 # systemLogger is our Vector image, configBuilder creates valid Cassandra configuration for
 # 3.11/4.0 and DSE 6.8, while k8ssandraClient is used to build configs for 4.1
 imageConfig:
-  systemLogger: k8ssandra/system-logger:v1.18.2
-  configBuilder: datastax/cass-config-builder:1.0-ubi7
-  k8ssandraClient: k8ssandra/k8ssandra-client:v0.2.2
+  systemLogger: cr.k8ssandra.io/k8ssandra/system-logger:v1.18.2
+  configBuilder: cr.dtsx.io/datastax/cass-config-builder:1.0-ubi7
```

Comment:
**Issue:** we'd need to get rid of the `-ubi7` here to get arm64 compatibility back and be consistent with what we do in k8ssandra-operator.

Reply:
I wonder if we can update the cass-config-builder to ubi8 or ubi9 with ARM support.

Reply:
@adejanovski Removing -ubi7 would be an issue as all these images are UBI images. cass-config-builder itself is an issue, but luckily entirely ignored for 4.1 and newer.

Reply:
We use `1.0.7` by default in k8ssandra-operator which isn't a ubi image and has arm support.

Reply:
Yes, but since it's not an UBI image, it's not usable for us.

Reply:
ok, let's resolve this then